aws/sqs/sqs_integration_test.go

```diff
@@ -182,6 +182,30 @@ func awsCmdCheckSQSAttribute(url, attribute, expectedValue string) bool {
 	return false
 }
 
+func TestCheckQueueExists(t *testing.T) {
+	// ARRANGE
+	setup()
+	defer teardown()
+
+	client, err := New()
+
+	// ASSERT
+	assert.Nil(t, err)
```

Comment:
Since the test isn't focused on whether client creation works, and subsequent client methods will fail if the client creation failed, this is more appropriate:

```go
client, err := New()
require.Nil(t, err, fmt.Sprintf("error creating sqs client: %v", err))
```